CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## v5.0.1 (Unreleased)
+## v5.0.1
 
 #### Bug fixes & Enhancements
+- [#127](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/127) Some integration tests are failing
```

Comment:
Really generic name on this, I'd incentivise to write a better, more descriptive title of the issue.